src/clients/stormGlass.ts

```diff
@@ -75,10 +77,30 @@ export class StormGlass {
     'swellDirection,swellHeight,swellPeriod,waveDirection,waveHeight,windDirection,windSpeed';
   readonly stormGlassAPISource = 'noaa';
 
-  constructor(protected request = new HTTPUtil.Request()) {}
+  constructor(
+    protected request = new HTTPUtil.Request(),
+    protected cacheUtil = new CacheUtil()
+  ) {}
 
-  //TODO someone to cache this call
   public async fetchPoints(lat: number, lng: number): Promise<ForecastPoint[]> {
+    const cacheForecastPoints = this.getForecastPointsFromCache(lat, lng);
```

Comment:
small nit here, I think `cachedForecastPoints` is better.

Reply:
Me too, I'll change it